<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_chiller</class>
 <widget class="QDialog" name="dialog_chiller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CHILLER</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_9">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 bold 9pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="title">
         <string>Temperature Out(C)</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="chiller_temp_label1">
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Sans Serif</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: bold 9pt &quot;Sans Serif&quot;;</string>
           </property>
           <property name="text">
            <string>HVPS1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="chiller_temp_1">
           <property name="styleSheet">
            <string notr="true">margin-top: 1px;
margin-bottom: 1px;
border-radius: 3px;
min-height: 20px;
min-width: 20px;
background-color: rgb(175, 175, 175);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="chiller_temp_label2">
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>HVPS2</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="chiller_temp_2">
           <property name="styleSheet">
            <string notr="true">margin-top: 1px;
margin-bottom: 1px;
border-radius: 3px;
min-height: 20px;
min-width: 20px;
background-color: rgb(175, 175, 175);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="chiller_temp_label3">
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>HVPS3</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="chiller_temp_3">
           <property name="styleSheet">
            <string notr="true">margin-top: 1px;
margin-bottom: 1px;
border-radius: 3px;
min-height: 20px;
min-width: 20px;
background-color: rgb(175, 175, 175);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_8">
        <property name="styleSheet">
         <string notr="true">font: 75 bold 9pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="title">
         <string>Flow Switch</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="4" rowspan="3">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="flow_sw_8">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>26</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
font: bold 8pt &quot;Sans Serif&quot;;
margin-top: 3px;
margin-bottom: 3px;
border-radius: 10px;
min-height: 20px;
min-width: 20px;
background-color: rgb(175, 175, 175);
}</string>
           </property>
           <property name="text">
            <string>8</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="flow_sw_5">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>26</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
font: bold 8pt &quot;Sans Serif&quot;;
margin-top: 3px;
margin-bottom: 3px;
border-radius: 10px;
min-height: 20px;
min-width: 20px;
background-color: rgb(175, 175, 175);
}</string>
           </property>
           <property name="text">
            <string>5</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="flow_sw_6">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>26</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
font: bold 8pt &quot;Sans Serif&quot;;
margin-top: 3px;
margin-bottom: 3px;
border-radius: 10px;
min-height: 20px;
min-width: 20px;
background-color: rgb(175, 175, 175);
}</string>
           </property>
           <property name="text">
            <string>6</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="flow_sw_4">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>26</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
font: bold 8pt &quot;Sans Serif&quot;;
margin-top: 3px;
margin-bottom: 3px;
border-radius: 10px;
min-height: 20px;
min-width: 20px;
background-color: rgb(175, 175, 175);
}</string>
           </property>
           <property name="text">
            <string>4</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QLabel" name="flow_sw_9">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>26</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
font: bold 8pt &quot;Sans Serif&quot;;
margin-top: 3px;
margin-bottom: 3px;
border-radius: 10px;
min-height: 20px;
min-width: 20px;
background-color: rgb(175, 175, 175);
}</string>
           </property>
           <property name="text">
            <string>9</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="flow_sw_3">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>26</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
font: bold 8pt &quot;Sans Serif&quot;;
margin-top: 3px;
margin-bottom: 3px;
border-radius: 10px;
min-height: 20px;
min-width: 20px;
background-color: rgb(175, 175, 175);
}</string>
           </property>
           <property name="text">
            <string>3</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QPushButton" name="pushButton">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>26</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
font: 75 bold italic 9pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 0);
margin-top: 3px;
margin-bottom: 3px;
border-radius: 10px;
min-height: 20px;
min-width: 20px;
}
</string>
           </property>
           <property name="text">
            <string>i</string>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="flow_sw_1">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>26</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
font: bold 8pt &quot;Sans Serif&quot;;
margin-top: 3px;
margin-bottom: 3px;
border-radius: 10px;
min-height: 20px;
min-width: 20px;
background-color: rgb(175, 175, 175);
}</string>
           </property>
           <property name="text">
            <string>1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="6" rowspan="3">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="flow_sw_2">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>26</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
font: bold 8pt &quot;Sans Serif&quot;;
margin-top: 3px;
margin-bottom: 3px;
border-radius: 10px;
min-height: 20px;
min-width: 20px;
background-color: rgb(175, 175, 175);
}</string>
           </property>
           <property name="text">
            <string>2</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="flow_sw_7">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>26</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
font: bold 8pt &quot;Sans Serif&quot;;
margin-top: 3px;
margin-bottom: 3px;
border-radius: 10px;
min-height: 20px;
min-width: 20px;
background-color: rgb(175, 175, 175);
}</string>
           </property>
           <property name="text">
            <string>7</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="3">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_10">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_28">
         <item row="0" column="1">
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 bold 9pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Chiller Interlock</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="chiller_interlock">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>26</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Sans Serif</family>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
font: bold 8pt &quot;Sans Serif&quot;;
margin-top: 3px;
margin-bottom: 3px;
border-radius: 10px;
min-height: 20px;
min-width: 20px;
background-color: rgb(175, 175, 175);
}</string>
           </property>
           <property name="text">
            <string>6</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
